Readability.js

```diff
@@ -346,8 +346,9 @@ Readability.prototype = {
     }
 
     curTitle = curTitle.trim();
-
-    if (curTitle.split(' ').length <= 4)
+    var curTitleLen = curTitle.split(' ').length;
+    if (curTitleLen <=4 && ( !origTitle.match(/ [\\\/>»] /)
+        || curTitleLen != origTitle.replace(/[\|\-\\\/>» ]+/g, " ").split(' ').length -1))
```

Comment:
> @ Your eslint config: no problem, but you could add a line to the test section of the readme file to make first-time users aware of eslint (I only found out when my pull request failed the  continuous-integration check)

An excellent point. I updated the "Contributing" and "Tests" sections of the ReadMe.